integreat_cms/static/src/js/pages/xliff-export-overlay.ts

```diff
@@ -0,0 +1,71 @@
+window.addEventListener("load", () => {
+  addXliffExportOverlayListeners(openXliffExportOverlay);
+  // Set listener for select toggles.
+  document.querySelectorAll(".bulk-select-language").forEach((el) => {
+    el.addEventListener("change", toggleXliffBulkActionButton); 
+  }); 
+  // Set listener for select-all toggle.
+  document.getElementById("select-all-languages")?.addEventListener("change", (toggle) => {
+    document.querySelectorAll(".bulk-select-language").forEach((el) => {
+      (el as HTMLInputElement).checked = (toggle.target as HTMLInputElement).checked; 
+      toggleXliffBulkActionButton(); 
+    });
+  });
+});
+
+
+function addXliffExportOverlayListeners(callback: (overlay: HTMLElement) => void) {
+  const overlay = document.getElementById("xliff_export_overlay");
+  if (overlay) {
+    // Set listener for bulk action execute button.
+    document.getElementById("bulk-action-execute")?.addEventListener("click", (event) => {
+      const xliffOptionIndex = (document.getElementById("xliff-multiple-languages-option") as HTMLOptionElement).index;
+      const selectedIndex = (document.getElementById("bulk-action") as HTMLSelectElement).selectedIndex; 
+      if (xliffOptionIndex === selectedIndex) {
+        event.preventDefault(); 
+        callback(overlay);
+      }
+    });
+    document.getElementById("btn-close-xliff-export").addEventListener("click", () => {
+      closeXliffExportOverlay(overlay);
+    });
+    // Close window by clicking on backdrop.
+    overlay.addEventListener("click", (e) => {
+      if (e.target === overlay) {
+        closeXliffExportOverlay(overlay);
+      }
+    });
+  }
+}
+
+
+function openXliffExportOverlay(overlay: HTMLElement) {
+  overlay.classList.remove("hidden");
+  overlay.classList.add("flex");
+}
+
+
+function closeXliffExportOverlay(overlay: HTMLElement) {
+  overlay.classList.add("hidden");
+  overlay.classList.remove("flex");
+  let selector = (document.getElementById("bulk-action") as HTMLSelectElement);
+  if (selector.hasAttribute("data-last-value")) {
+    selector.value = selector.getAttribute("data-last-value");
+    selector.setAttribute("data-value", selector.value); 
+  }
+  toggleXliffBulkActionButton();
+}
+
+
+function toggleXliffBulkActionButton() {
+  // Only activate button if at least one language item is selected
+  const selectLanguages = <HTMLInputElement[]>(
+    Array.from(document.getElementsByClassName("bulk-select-language"))
+  )
+  const multiLanguageXliffButton = document.getElementById("xliff-overlay-bulk-action-execute") as HTMLButtonElement; 
+  if (selectLanguages.some((el) => el.checked)) {
+    multiLanguageXliffButton.disabled = false;
+  } else {
+    multiLanguageXliffButton.disabled = true; 
+  }
```

Comment:
```suggestion
  multiLanguageXliffButton.disabled = !selectLanguages.some((el) => el.checked);
```
This expression can be simplified.